src/test/java/co/rsk/bitcoinj/core/StoredBlockTest.java

```diff
@@ -1,44 +1,77 @@
 package co.rsk.bitcoinj.core;
 
 import org.junit.Before;
+import org.junit.Rule;
 import org.junit.Test;
+import org.junit.rules.ExpectedException;
 import org.spongycastle.util.encoders.Hex;
 
 import java.math.BigInteger;
 import java.nio.ByteBuffer;
+
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
```

Comment:
nit: move the static imports to the top of the file

Reply:
Updated